src/plugins/home/server/services/sample_data/sample_data_registry.ts

```diff
@@ -51,7 +51,7 @@ export class SampleDataRegistry {
       makeSampleDataUsageCollector(usageCollections, this.initContext);
     }
     const usageTracker = usage(
-      core.savedObjects,
+      core.getStartServices().then(([coreStart]) => coreStart.savedObjects),
       this.initContext.logger.get('sample_data', 'telemetry')
     );
```

Comment:
Another approach here: Instead of passing a reference to `CoreSetup.getStartServices`, I create the `savedObjectStart` promise outside and pass it to the function. Not sure if this is better or worse.